wear/src/main/java/info/nightscout/androidaps/interaction/actions/ViewSelectorActivity.java

```diff
@@ -59,7 +59,12 @@ public void setAdapter(GridPagerAdapter adapter) {
         pager.setAdapter(adapter);
     }
 
-    private void setTitleBasedOnScreenShape(String title) {
+    private void setTitleBasedOnScreenShape() {
+        // intents can inject dynamic titles, otherwise we'll use the default
+        String title = String.valueOf(getTitle());
+        if (getIntent().getExtras() != null)
+            title = getIntent().getExtras().getString("title", title);
```

Comment:
Prefer if statement without prentices, please use a single line, or add the conditional section {}